src/components/Canvas.vue

```diff
@@ -44,20 +44,40 @@ import PNode from '../components/Node'
 import PEdge from '../components/Edge'
 
 import Number from '../components/nodes/Number'
+import Boolean from '../components/nodes/Boolean'
+import String from '../components/nodes/String'
+import File from '../components/nodes/File'
 import Average from '../components/nodes/Average'
 import Binarize from '../components/nodes/Binarize'
 import Note from '../components/nodes/Note'
+import Addition from '../components/nodes/Addition'
+import Subtraction from '../components/nodes/Subtraction'
+import Multiplication from '../components/nodes/Multiplication'
+import Division from '../components/nodes/Division'
+import Exponentiation from '../components/nodes/Exponentiation'
+import Root from '../components/nodes/Root'
+import Logarithm from '../components/nodes/Logarithm'
```

Comment:
Please prefix each import name with some prefix e.g. `PN`. For example `import PNText from '../components/nodes/Text`.